src/pages/ServiceJaeger/ServiceJaegerPage.tsx

```diff
@@ -28,10 +28,14 @@ const EmptyStatePage = () => (
 class ServiceJaegerPage extends React.Component<RouteComponentProps<{}>, ServiceJaegerState> {
   constructor(props: RouteComponentProps<{}>) {
     super(props);
-    this.state = { jaegerURL: '', error: false };
+    this.state = {
```

Comment:
Is the jaegerURL really stateful? Do we switch jaegerURLs in the application and expect a render?

Reply:
No @mtho11, this will be like before we request to the backend for the Jaeger url, this is a PoC